CHANGELOG.md

```diff
@@ -6,7 +6,8 @@ All notable changes to this project will be documented in this file, in reverse
 
 ### Added
 
-- Nothing.
+- Default behavior changed when malformed is pushed into toUtf8.  Rather than dumping the bad string in an exception (dangerous behavior), it will return a blank string.
+- Boolean 'debug' parameter added as second constructor argument, that preserves the old behavior when set to true. 
```

Comment:
No, please remove this. If you want to run an escaper in debug mode, subclass the `Escaper` and implement something like:

```php
class DebugEscaper extends \Zend\Escaper\Escaper
{
    public function escapeHtml($toEscape) { return 'ESCAPED-HTML-START' . $toEscape . 'ESCAPED-HTML-END'; }
}